functest/send_money_test.go

```diff
@@ -84,7 +86,7 @@ func TestTransferMoneyToNotExist(t *testing.T) {
 
 	_, err := signedRequest(firstMember, "member.transfer", map[string]interface{}{"amount": amount, "toMemberReference": testutils.RandomRef().String()})
 	require.NotNil(t, err)
-	require.Contains(t, err.Error(), "index not found")
+	require.True(t, errors.IsNonRetryable(err), "index not found")
```

Comment:
The last param of `require.True` is the additional message to tests trace in case of fail. Why is it says `index not found`?

Reply:
We send API request here, so you don't get direct instance of `Retryable` error, you get some json with `error` field inside, and convert it to basic `error` interface for working with it like its object.